libyara/modules.c

```diff
@@ -191,3 +191,9 @@ int yr_modules_unload_all(YR_SCAN_CONTEXT* context)
 
   return ERROR_SUCCESS;
 }
+
+void yr_modules_print_names(void)
```

Comment:
I think we should make this a proper API that returns the list of module names without printing it. You may want to get the list of modules for some other reason besides printing it to the console.

Reply:
Done, but I'm not sure you will like the name I went with: `yr_modules_get_table()`. I'm happy to rename it to something better if you have suggestions, maybe `yr_modules_get_modules()`?

Also, I'll update docs for this before merging.

Lastly, any thoughts on adding something like this to yara-python? I think it would be useful to be able to get that information from there too. I'll add it there if you agree.

Reply:
Yes, I think that adding it to yara-python may be useful too.